tests/e2e-integration/e2e/sections/collection/Collection.spec.ts

```diff
@@ -42,7 +42,9 @@ describe('Collection Page', () => {
       cy.findByText('New Dataset').should('be.visible').click({ force: true })
     })
     cy.wait(1000)
-    cy.findByText(/Create Dataset/i).should('exist')
+    cy.get(`.breadcrumb`)
```

Comment:
You can discard these assertions of the breadcrumb in this test as they are already being asserted in the CreateDataset spec